Sources/FoundationEssentials/FileManager/FileManager+Files.swift

```diff
@@ -907,6 +907,7 @@ extension _FileManagerImpl {
             let group = attributes[.groupOwnerAccountName] as? String
             let groupID = _readFileAttributePrimitive(attributes[.groupOwnerAccountID], as: UInt.self)
             
+            #if !os(WASI) // WASI does not have the concept of users or groups
```

Comment:
Should we throw a `.featureUnsupported` error if the caller tries to provide these attribute values?

Reply:
Ah, yes, we should throw in that case.